### ISSUE TITLE
train problem

### ISSUE THREAD

**Issue by Sheffy001** [HighwayWu/ImageInpainting#17]
Why do I get the error weight1, indexes1 = torch.topk(cosine1, k, dim=2) at 253 when using my training set of 5000 images, but it trains normally with 252 images? What could be the reason for this?
